presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriverUri.java

```diff
@@ -39,26 +66,29 @@
 
     private final HostAndPort address;
     private final URI uri;
+    private final Properties properties;
 
     private String catalog;
     private String schema;
 
     private final boolean useSecureConnection;
 
-    public PrestoDriverUri(String url)
+    public PrestoDriverUri(String url, Properties driverProperties)
```

Comment:
I would rather rename this class to something like `PrestoConnectionConfiguration`